File: Trivia/SendQuestionResponse.cpp
#include "SendQuestionResponse.h"

SendQuestionResponse::SendQuestionResponse(std::string question, std::vector<std::string> answers, unsigned int remaining) : _question(question), _answers(answers), _remaining(remaining)
{
}

std::string SendQuestionResponse::getQuestion() const
{
	return _question;
}

std::vector<std::string> SendQuestionResponse::getAnswers() const
{
	return _answers;
}

unsigned int SendQuestionResponse::getRemainingQuestions() const
{
	return _remaining;
}

File: TriviaClient/Responses/SendQuestionResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TriviaClient.Responses
{
    class SendQuestionResponse
    {
        string _question;
        string[] _answer;
        uint _remaining;

        SendQuestionResponse(string question, string[] answers, uint remaining)
        {
            _question = question;
            _answer = answers;
            _remaining = remaining;
        }

        public string GetQuestion()
        {
            return _question;
        }

        public string[] GetAnswers()
        {
            return _answer;
        }
    }
}
